Update dependencies from https://github.com/dotnet/arcade build 20230…

nohwnd opened the pull request:
According to https://github.com/dotnet/arcade/issues/13648 we need to update VS to 17.6, is this the way to do it? 

Merging changes from #4470.

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/dotnet/arcade build 20230519.2

Microsoft.DotNet.Arcade.Sdk
 From Version 8.0.0-beta.23262.5 -> To Version 8.0.0-beta.23269.2

Dependency coherency updates

Microsoft.DotNet.XliffTasks,Microsoft.SourceLink.GitHub
 From Version 1.0.0-beta.23261.1 -> To Version 1.0.0-beta.23266.1 (parent: Microsoft.DotNet.Arcade.Sdk

nohwnd committed:
Update required VS to 17.6

nohwnd committed:
Merge branch 'darc-main-194ec78c-3f53-4b2e-bb7f-6968e1e72b6e' into update-vs

nohwnd commented:
Let's see if this builds, or if some other variable needs updating. Or the VS on disk.

Evangelink reviewed (commented):
Thanks for testing this @nohwnd. It's also unclear to me what's the needed change. If we can't figure it out, I will ask on the related arcade ticket.

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/dotnet/arcade build 20230529.1

Microsoft.DotNet.Arcade.Sdk
 From Version 8.0.0-beta.23262.5 -> To Version 8.0.0-beta.23279.1

Dependency coherency updates

Microsoft.DotNet.XliffTasks,Microsoft.SourceLink.GitHub
 From Version 1.0.0-beta.23261.1 -> To Version 1.0.0-beta.23272.1 (parent: Microsoft.DotNet.Arcade.Sdk

nohwnd committed:
Merge branch 'darc-main-194ec78c-3f53-4b2e-bb7f-6968e1e72b6e' into update-vs

nohwnd committed:
Revert global.json changes

nohwnd commented:
waiting for the build agent to have 17.6, then the original PR should be mergeable.
